### PR TITLE
Detect the tag iblaze_notify_changes and use NotifyCommand instead

### PR DESCRIPTION
**Pull request by achew22**

### CODE REVIEW

File: ibazel/BUILD [hunk 1]
```diff
@@ -18,6 +18,9 @@ go_binary(
     name = "ibazel",
     importpath = "github.com/bazelbuild/bazel-watcher/ibazel",
     library = ":go_default_library",
+    tags = [
+        "IBAZEL_MAGIC_TAG",
```
Comment (mrmeku) [inline ibazel/BUILD]: Why is this here? Can you leave a comment saying why this is necessary?

File: third_party/bazel/master/src/main/protobuf/build.proto [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// This file contains the protocol buffer representation of a build
+// file or 'blaze query --output=proto' call.
+
```
Comment (mrmeku) [inline third_party/bazel/master/src/main/protobuf/build.proto]: Do you want to say where this proto was sourced from?

Comment (achew22) [reply third_party/bazel/master/src/main/protobuf/build.proto]: Done and added some more tests
